control-plane/pkg/reconciler/consumer/consumer.go

```diff
@@ -204,15 +209,6 @@ func (r Reconciler) reconcileAuth(ctx context.Context, c *kafkainternals.Consume
 				Version:   secret.ResourceVersion,
 			},
 		}
-		ref := tracker.Reference{
-			APIVersion: c.APIVersion,
-			Kind:       c.Kind,
-			Namespace:  c.GetNamespace(),
-			Name:       c.GetName(),
```

Comment:
hmm why was this Reference tracker even here before?

Reply:
previously, we were only tracking the case where there is only one secret reference (all cases except the KafkaSource one), we're now tracking properly both cases.